src/docs/product/alerts-notifications/alert-settings.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Alert Settings
+sidebar_order: 3
+---
+
+Configure alerts to your team's needs with alert rules and settings. Create new alerts, view alerts, and designate the recipient(s) of each alert.
+
+## Rules
+
+Alert Rules shows all of your project's alerts, their type, name, triggers, and actions. Use this page to create new alerts, determine if a component of your project needs alerting, or recognize unnecessary alerts.
+
+![Alert rules are divided by projects, their conditions/triggers, and actions(s).](alert-listing.png)
+
+## Settings
+
+Alerts have default settings and options to customize alerts to your personal needs.
+
+### Alert Settings
+
+Alert settings can only be edited by users with the organization owner, manager, or admin role. With alert settings you can configure alert email templates and set minimum/maximum delivery intervals for alerts. Access alert settings by navigating to **Settings > [Organization] > [Project] > Alerts**.
+
+#### Digests
```

Comment:
@adhiraj I moved Digests here and I think it makes sense because the digest feature is a part of Alert Settings. Let me know if there's a better place for this.

Reply:
Link to this from Issue Alerts